src/main/kotlin/BdvNotifier.kt

```diff
@@ -2,6 +2,7 @@ package org.mastodon.mamut
 
 import bdv.viewer.TimePointListener
 import bdv.viewer.TransformListener
+import graphics.scenery.utils.lazyLogger
```

Comment:
only a remark (from somebody who doesn't know what/how scenery's lazyLogger is/functions....)

Is lazyLogger anyhow compatible with Fiji's logging subsystem or a completely separate thing?

The Fiji-way of doing it, would be:

- obtain `LogService`
- ask it to create a `Logger` dedicated to the session/programe
- and log anywhere via this logger with `info(msg)` or `debug()` or `error`, `warn`... the usual stuff

And.. there's happens to be such a thing already going on:
- [LogService is obtained from the scijava plugin that Mastodon starts](https://github.com/xulman/mastodon-sciview-take2/blob/e9cf03a17702e3a956ffb38e02ad211e80d53fb2/src/main/kotlin/plugins/scijava/MastodonSidePlugin.kt#L60)
- it's missing `Logger log = logService.subLogger()` [somewhere around here](https://github.com/xulman/mastodon-sciview-take2/blob/e9cf03a17702e3a956ffb38e02ad211e80d53fb2/src/main/kotlin/plugins/scijava/MastodonSidePlugin.kt#L105)
- which could be [passed into the MSB](https://github.com/xulman/mastodon-sciview-take2/blob/e9cf03a17702e3a956ffb38e02ad211e80d53fb2/src/main/kotlin/plugins/scijava/MastodonSidePlugin.kt#L112)

Reply:
but the example above is pretty much all from the *official proper Fiji logging* ever used in the project (which is why you are adding the proper logging now)

that said, we're not mixing up two log systems (Fiji's and `lazyLogger`) as there's nearly nothing used from the Fiji one

Reply:
I wasn't aware of the Logservice, but I know that scenery always uses lazylogger for everythhing and I think sciview also does, so I just went with that.

Reply:
no worries, important is that *exactly* one log service is used, which is the case now;
so all is just great :-)